astroquery/mast/collections.py

```diff
@@ -42,6 +42,7 @@ def __init__(self):
 
         services = {"panstarrs": {"path": "panstarrs/{data_release}/{table}.json",
                                   "args": {"data_release": "dr2", "table": "mean"}}}
+
```

Comment:
Now I'm confused: at some iterations plato was added to this one, too, but then I don't see it's being used anywhere. So what is this attribute used for, and why is `panstarrs` the only element in there?

Reply:
When I first started this PR, I added `plato` to the services call before realizing I didn't need to because plato was in the Mashup API already, so I removed it from this services call a few commits later. I think what's happening is if you're looking at individual commits instead of `Files Changed`, you're seeing a version of this branch where I had `plato` added, but I removed it later down the line.

`panstarrs` has its own special `services` call because `panstarrs` is not part of the Mashup API. It's only part of the Catalogs service. So that's why you'll see a bunch of `if, else` statements where, if the catalog is indeed `panstarrs`, the database gets called from another location using `service_api_connection` instead of `portal_api_connection`. See example here: Lines 111 to 115 https://github.com/astropy/astroquery/blob/5740dc772a1924e3446b85bbf898d4f9f8fc3ea7/astroquery/mast/collections.py#L111

Reply:
> panstarrs has its own special services call because panstarrs is not part of the Mashup API.

OK, fine, this is the info I was after, that panstarrs is actually the odd one out.

Reply:
btw, I made this comment on the overall diff rather than on the commit level, yet it brings it up as I had made it on the first commit. So I indeed think it would be useful to squash all together to remove that temporary change in the middle. After all we occasionally go back to the commit level.

Reply:
Sure thing!